spec/services/default_shipping_category_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe DefaultShippingCategory do
+  describe '.create!' do
+    it "names the location 'Default'" do
+      shipping_category = described_class.create!
+      expect(shipping_category.name).to eq('Default')
```

Comment:
It's not a big deal, but there's lots of unnecessary parentheses in this file. You can use `eq 'Default'` here, for example. I'm surprised`rubocop` is not complaining. I'm not sure what IDE or editor you're using, but many have `rubocop` plugins that add tips for code style, and automated linting.

Reply:
I have the Rubocop linter added to Sublime 3 - it wasn't catching these, but I'll check the style guide to make sure that my settings are correct. Thanks!